src/cpp/include/openvino/genai/speculative_decoding/perf_metrics.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/genai/perf_metrics.hpp"
+#include "openvino/genai/visibility.hpp"
+
+
+namespace ov::genai {
+
+struct OPENVINO_GENAI_EXPORTS SDPerfMetrics : public ov::genai::PerfMetrics {
+    ov::genai::MeanStdPair ttst;  // Time to the second token (in ms).
+    ov::genai::MeanStdPair ttl;  // latency from the third token (in ms).
```

Comment:
Add a comment why the third token is introduced.

Is there the same strangeness for the second token for greedy?
Does it hold for CPU and GPU?
You'll probably need to summon the plugins once you answerer these questions.